konnect-runtime-setup.sh

```diff
@@ -13,6 +13,10 @@ KONNECT_TP_ENDPOINT=
 KONG_CLUSTER_KEY_FILENAME=
 KONG_CLUSTER_CERT_FILENAME=
 
+VERSIONS_TO_APPLY_ALPINE_SUFFIX=(
+    "kong/kong-gateway:3.0.0.0"
+    )
```

Comment:
nit: spacing is off and since this is now moved away from the conditional we should add a comment explaining this.

```suggestion
# array of Kong Gateway versions which require Alpine tag for multi-platform support
VERSIONS_TO_APPLY_ALPINE_SUFFIX=(
  "kong/kong-gateway:3.0.0.0"
)
```